.nvmrc

```diff
@@ -1 +1 @@
-16
+>=16.0.0
```

Comment:
Is this change related to the issue?

Reply:
No, it was auto-merged like this - work on this issue has started and its perfectly ok that there are several issues that needs to be addressed.
You'll be notified when the issue i ready for inspection.  :)